### PR TITLE
Update `docmain` for pytket 1.30 release

### PR DESCRIPTION
**Pull request by CalMacCQ**
# Description

Updating `docmain` from the latest `main` for the pytket 1.30.0 release.

### CODE REVIEW

File: pytket/docs/conf.py [hunk 1]
```diff
@@ -43,7 +43,6 @@
 release = "1.30.0"
 
 
-
```
Comment (cqc-melf) [inline pytket/docs/conf.py:45]: are this changes that are not on main?

**Review by cqc-melf — COMMENTED** [review]
Might be better to resolve the issues with black on main and not here, because that would cause further issues next time? Is the docmain branch protected? It might be good to just force push the current state of main?

**Comment by CalMacCQ** [comment]
> Might be better to resolve the issues with black on main and not here, because that would cause further issues next time? Is the docmain branch protected? It might be good to just force push the current state of main?

Yeah the `docmain` branch is protected.

**Comment by cqc-melf** [comment]
There are still differences between the update branch and `main` is that intended? https://github.com/CQCL/tket/compare/main...update_docs/1.30

**Pull request closed without merging**